### ISSUE TITLE
[a11y] Cannot add attachments due to 'fieldName' error

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1423]
![image](https://user-images.githubusercontent.com/37256050/167207492-48cad4b8-82e1-4c92-a234-e5848b4387b5.png)

Cannot add attachments to CO records on `nm3`

https://a11y-2-2.test.specifysystems.org/specify/view/collectionobject/55327/?recordsetid=4

username: spadmin
